content/articles/eof.md

```diff
@@ -0,0 +1,181 @@
+---
+author_name: "Jonah Bossewitch"
+author_url: http://alchemicalmusings.org/about
+date: 2016-09-12
+lede: "Dr. J's farewell post."
+poster: poster-bossewitch-pmt-inactive.jpg
+poster_source: ""
+poster_sourceurl: ""
+tags: ["collaboration", "designresearch", "opensource", "freeculture"]
+title: "EOF"
+topics:
+- Projects
+type: post
+---
+
+This is my final post on this blog as a Columbia employee. Taking a cue from
+our not-so-lame-duck prez (see:
+[#obamajama](http://jama.jamanetwork.com/article.aspx?articleid=2533698
+#obamajama)), I’ll indulge in the conceit of reflecting on my legacy at
```

Comment:
It looks like this link: `[#obamajama](http://jama.jamanetwork.com/article.aspx?articleid=2533698#obamajama))` needs to be on a single line to fix formatting.